evap/static/scss/components/_modal.scss

```
@@ -2,6 +2,28 @@
     max-width: 1120px;
 }
 
+.modal-grid {
+    display: grid;
+    width: 75%;
+    margin: 1.5rem auto 1.5rem auto;
```

I think this is equivalent to
```suggestion
    margin: 1.5rem auto;
```